dev_scripts_helpers/github/sync_gh_projects.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env python3
+"""
+Synchronize the structure (fields and views) of a GitHub Projects Beta board
+from a template project to a destination project, without touching title or
+content.
+
+Example:
+    ./sync_gh_project_structure.py \
+      --src-template "[TEMPLATE] Causify Project" \
+      --dst-project "Buildmeister"
+
+Helper: get_project_titles() returns all project titles for quick lookup.
+"""
+
+import argparse
+import json
+import logging
+import subprocess
+import sys
+
+DEFAULT_OWNER = "causify-ai"
+_LOG = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO, format="%(levelname)s: %(message)s")
+
+
+def _run_command(cmd: list) -> str:
+    """
+    Run a command and return its stdout or exit on error.
+    """
+    _LOG.debug("Running command: %s", " ".join(cmd))
+    result = subprocess.run(cmd, capture_output=True, text=True)
+    if result.returncode != 0:
+        _LOG.error(
+            "Error running command:\n%s\n%s", " ".join(cmd), result.stderr.strip()
+        )
+        sys.exit(1)
+    return result.stdout
+
+
+def _get_projects() -> list:
+    """
+    Retrieve up to 100 Projects Beta boards for DEFAULT_OWNER (including
+    closed).
+
+    Returns list of dicts with 'title', 'id', and 'number'.
+    """
+    cmd = [
+        "gh",
```

Comment:
let's make it more readable

cmd = [
   "gh poject list",
   f"--owner {...}"